File: app/src/main/java/com/example/review/DataStructureFile/ElementCategory.java
package com.example.review.DataStructureFile;


import android.support.annotation.NonNull;

public class ElementCategory {
    public String   txt;
    public Category category;

    public ElementCategory() {
    }

    public ElementCategory(String txt, Category category) {
        this.txt = txt;
        this.category = category;
    }

    @NonNull
    @Override
    public String toString() {
        return txt + " - ( " + category.toString() + " )";
    }

    public enum Category {
        correct,
        malposition,
        unnecesary,
        missing
    }
}

File: app/src/main/java/com/example/review/DataStructureFile/ElementCategory.kt
package com.example.review.DataStructureFile

class ElementCategory {
    var txt: String? = null
    var category: Category? = null

    constructor() {}
    constructor(txt: String?, category: Category?) {
        this.txt = txt
        this.category = category
    }

    override fun toString(): String {
        return txt + " - ( " + category.toString() + " )"
    }

    enum class Category {
        correct, malposition, unnecesary, missing
    }
}